docs/pages/getting-started.html

```diff
@@ -50,7 +50,11 @@ <h3 class="usa-heading">For developers</h3>
 
 <h4>Browser support:</h4>
 
-<p>The standards are designed to support both older and newer browsers through progressive enhancement, and officially support Internet Explorer 9 and up, and the latest versions of Chrome, Firefox, and Safari. Internet Explorer 8 and below generally see very low usage today, and most agency websites should be able to safely begin support at Internet Explorer 9.</p>
+<p>The Standards are designed to support both older and newer browsers through progressive enhancement. They officially support IE 9 and up, along with the most recent versions of Chrome, Firefox, and Safari. Internet Explorer 8 and below see very low usage, and most agency websites should be able to safely begin support at Internet Explorer 9.</p>
+
+<h4>Progressive enhancement:</h4>
+
+<p>The Standards embrace progressive enhancement, which is a web development strategy with accessibility and responsiveness at its heart. Server-side languages and frameworks are leveraged for the majority of <strong>HTML</strong> generation. These pages are enhanced with CSS to create a rich user experience and design. Finally, client-side languages (like JavaScript) are applied to add the final layer of enhancement to the experience without losing the underlying functionality of the page or site. This means most processes are run on the servers hosting the site, meaning they aren't dependent on the client. For many users, this offers substantial benefits — mobile devices typically have fewer resources than desktops or laptops, for instance, and progressive enhancement offers mobile users a better experience all around. Further, you can focus on the functionality of the site, ensuring it works properly (the expected HTML is delivered) before continuing to enhance the experience. Finally, by maximizing server-side processing, you minimize the negative impact on battery life client-side processing can have.</p>
```

Comment:
@maya — I also agree that removing mentions of client-side servers would clarify this definition.